test/JDBC/expected/TestTriggerInteroperability.out

```diff
@@ -0,0 +1,170 @@
+-- tsql
+create table test_tbl_trig_bbf_1(a int);
+create table test_tbl_trig_bbf_2(a int);
+GO
+
+-- psql
```

Comment:
we want to add these test cases with and without error throw (ignore if they are already part of TestProcedureWithTriggers.mix)
begin TSQL insert -> TSQL trigger -> PG trigger (error)
begin TSQL insert -> PG trigger -> TSQL trigger -> PG trigger (error)
begin TSQL insert -> TSQL trigger -> PG trigger -> TSQL trigger (error)